lisa/node.py

```diff
@@ -102,9 +103,10 @@ def support_sudo(self) -> bool:
         # check if sudo supported
         if self.is_posix and self._support_sudo is None:
             process = self._execute("command -v sudo", shell=True, no_info_log=True)
-            result = process.wait_result(10)
+            result = process.wait_result(5)
```

Comment:
Let's keep 10 seconds here for slowing environments or connections.

Reply:
Done